src/pe_reports/helpers/bulletin/bulletin_generator.py

```diff
@@ -0,0 +1,274 @@
+"""
+Bulletin/bulletin_generator: A tool for creating Posture & Exposure Bulletins.
+
+Usage:
+  bulletin_generator INTEL_ITEM_ID
+
+Options:
+  -h --help                         Show this message.
+  INTEL_ITEM_ID                     Cyber Sixgill intel item id
+"""
+
+# Standard Python Libraries
+import datetime
+import logging
+import os
+
+# Third-Party Libraries
+from docopt import docopt
+import jinja2
+import pandas as pd
+import pdfkit
+
+# cisagov Libraries
+from pe_reports.data.db_query import connect
+from pe_source.data.pe_db.config import cybersix_token
+from pe_source.data.sixgill.api import intel_post
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_post(id):
+    """Retrieve a cybersix post based on the intel item id."""
+    query = f"_id:{id}"
+    token = cybersix_token()
+    resp = intel_post(token, query, frm=0, scroll=False, result_size=1)
+    return resp
+
+
+def date_format(value, format="%m/%d/%Y"):
+    """Format a date field."""
+    val = datetime.datetime.strptime(value, "%Y-%m-%dT%H:%M:%S")
+    return val.strftime(format)
+
+
+def html_builder(text):
+    """Build out an html string for command line usage."""
+    input_type = input(
+        "Which of the following would you like to insert:\n [P]aragraph\n [B]ulleted List \n [N]umbered List\n Please provide a selection:"
+    )
+    if input_type == "P":
+        LOGGER.info("Paragraph Selected")
+        paragraph = input("Please enter paragraph text:")
+        paragraph = f"<p> {paragraph} </p>"
+        text = text + f"\n {paragraph}"
+
+    elif input_type == "B":
+        LOGGER.info("Bulleted List Selected. Enter [D] when done.")
+        bullets = "<ul>\n"
+        while True:
+            item = input("Enter line item: ")
+            if item == "D":
+                bullets = bullets + "</ul>"
+                break
+            bullets = bullets + f"<li>{item}</li>\n"
+        text = text + f"\n {bullets}"
+
+    elif input_type == "N":
+        LOGGER.info("Numbered List Selected")
+        bullets = "<ol>\n"
+        while True:
+            item = input("Enter line item: ")
+            if item == "D":
+                bullets = bullets + "</ol>"
+                break
+            bullets = bullets + f"<li>{item}</li>\n"
+        text = text + f"\n {bullets}"
+    else:
+        LOGGER.info("Invalid Selection")
+
+    cont = input("Would you like to add more content (Y/N): ")
+    if cont == "Y":
+        text = html_builder(text)
+
+    return text
+
+
+bulletin_path = os.path.dirname(os.path.realpath(__file__))
+
+
+def generate_cybersix_bulletin(
+    id,
```

Comment:
@DJensen94 , 
Missing doc strings at def generate_cybersix_bulletin:. Move doc string from below line 95.

Reply:
The doc string is on line 96 because the function call includes these parameters so I think they should be on line 96